waifu2x.ts

```diff
@@ -81,6 +83,9 @@ export interface Waifu2xVideoOptions extends Waifu2xOptions {
 }
 
 export default class Waifu2x {
+
+    static processes: child_process.ChildProcess[] = []

```

Comment:
I prefer that you import the `ChildProcess` type instead of accessing it from `child_process`.